api-reference/10 UI Components/dxCalendar/1 Configuration/weekNumberRule.md

```diff
@@ -6,7 +6,196 @@ default: 'auto'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a [week number](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#showWeekNumbers) calculation rule. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Calendar/Overview/"
+}
+
+This property can take on one of the following values:
+
+- `firstDay`    
+The first week of the year is the week that contains January 1. 
```

Comment:
It does not important which year is this. It isn't necessarily the "current" year.

```suggestion
The first week of a year is the week that contains January 1. 
```